mods/d2k/rules/aircraft.yaml

```diff
@@ -72,16 +49,17 @@ frigate:
 		Name: Frigate
 		Description: Supply spacecraft
 	Plane:
-		ROT: 5
-		Speed: 350
-		RepairBuildings: repair
+		CruiseAltitude: 1800
+		ROT: 7
+		Speed: 296	##148
```

Comment:
Well I really don't want default speed, it's not fun. There could always be a compromise of something in the middle, 1.5x or 1.75x the speed but I'm definitely not doing default.